src/sections/Question/Question.module.scss

```diff
@@ -0,0 +1,80 @@
+@use '../../styles/mixins.module';
+@use '../../styles/colors.module';
+
+.content {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-gap: 2rem;
+
+  @include mixins.media(laptop) {
+    grid-template-columns: 1fr;
+    grid-gap: 0;
+    justify-items: center;
+
+    & .sectionText {
+      text-align: center;
+    }
+  }
+}
+
+.inputContainer {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-gap: 1rem;
+  padding-bottom: 1rem;
+
+  @include mixins.media(laptop) {
+    display: block;
+  }
+}
+
+.message {
+  grid-column: span 2;
+}
+
+form {
+  margin-top: 2.75rem;
+  box-sizing: border-box;
+  align-items: center;
+  border-radius: 0.375rem;
+  border: solid colors.$primary-3 1px;
+  padding: 1rem 2rem;
+
+  @include mixins.media(laptop) {
+    width: 60%;
+  }
+
+  @include mixins.media(mobile) {
+    width: 100%;
+  }
+
+  & button {
+    margin-left: auto;
+  }
+}
+
+.input {
+  margin-bottom: 1rem;
+  align-content: flex-start;
+  gap: 0;
```

Comment:
I don't think this is needed. Also gap isn't supported very well with safari so I would avoid this

Reply:
This is part of the hacky fix I was talking about, for this issue:
<img width="323" alt="Screen Shot 2022-05-10 at 12 24 10 AM" src="https://user-images.githubusercontent.com/34602868/168528499-419e7020-d92b-45cd-b458-5c0c4a60aab4.png">
<img width="336" alt="Screen Shot 2022-05-10 at 12 24 06 AM" src="https://user-images.githubusercontent.com/34602868/168528501-faa2c475-fd0f-44e8-a2dc-f5e01c9b8dd8.png">
(When the error text is displayed, the question box moves down slightly.)
If you have better fixes, I'm open to suggestions.

Reply:
We can toggle a class to get rid of the gap (if not laptop single column) only if the input has an error